### PR TITLE
[MU4] Preferences. MIDI device mapping

### PR DESCRIPTION
**Pull request by RomanPudashkin**
midi-device-mappings

### CODE REVIEW

File: src/framework/ui/internal/uiconfiguration.cpp [hunk 1]
```diff
@@ -266,15 +266,44 @@ void UiConfiguration::writeThemes(const ThemeList& themes)
...
     return db.families();
 }
 
-ThemeList UiConfiguration::themes() const
-{
-    return m_themes;
+QStringList UiConfiguration::possibleAccentColors() const
+{
+    static const QStringList lightAccentColors {
+        "#F36565",
```
Comment (cbjeukendrup) [inline src/framework/ui/internal/uiconfiguration.cpp:283]: Couldn't these be part of the `ThemeInfo` struct? (either directly in the struct or in the `QMap<ThemeStyleKey, QVariant> values`)

**Comment by igorkorsukov** [comment]
@Tantacrul  by the way, interesting, why this names "MIDI Device Mapping", but not, for example, "MIDI Control" or "Control by MIDI"

**Comment by Tantacrul** [comment]
> @Tantacrul by the way, interesting, why this names "MIDI Device Mapping", but not, for example, "MIDI Control" or "Control by MIDI"

It could be called 'MIDI Control'. In fact that could be a better name.

MIDI Device Mapping is more precise... but I'm also worried it might sound a bit more advanced than it really is.

**Comment by Tantacrul** [comment]
BTW: It would be 'Mapping', not 'Mappings'

File: src/framework/ui/internal/uiconfiguration.cpp (hunk 1 above)
Comment (RomanPudashkin) [reply src/framework/ui/internal/uiconfiguration.cpp:288]: I think ThemeInfo should contain only values that affect the UI. Those values are used only in the accent colors list and they don't affect to color of any UI element, so better don't store them in the theme

**Review by Eism — APPROVED** [review]

**Review by Eism — APPROVED** [review]

**Pull request merged**